### DISCUSSION TITLE
Bunrodea Requirements

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9210] [Q&A] [answered]
I recently went to Bunrodea space and read on the wiki that you had to engage in battle with some 'Erabu Royal Guards' and head to human space to access their worlds. I destroyed one guard ship and went to the nearest planet (Sunracer or Canyon), but nothing popped up. Did I mess up, or is the quest not finished yet?

**Comment by lumbar527** [comment, marked as answer]
https://github.com/endless-sky/endless-sky/blob/fd241fe97010956ae0c34cfb1deb0befb8b3320e/data/bunrodea/bunrodea%20missions.txt#L24-L30

It won't always show up immediately; try taking off and landing a bunch. Or just go about doing stuff; it should come eventually.
